app/src/main/java/com/xmartlabs/daydreaming/ui/CustomScreenFragment.java

```diff
@@ -21,12 +24,16 @@
  */
 @FragmentWithArgs
 public class CustomScreenFragment extends BaseFragment {
+  public static final int MAX = 100;
```

Comment:
I agree with @diegomedina248 that it's not clear what it does when you read it. When you read the var declaration it should be somewhat clear what the var is for. Perhaps this shouldn't be a constant and just be used in code.

And I think it should be private, not public.